test.cpp

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include "quadro_solve.cpp"
+#define ANSWER_SIZE 256
+
+/**
+    \brief The main function that imports quadro_solve and displays the value of each of the roots
+*/
+
+int main()
+{
+    double a = 0, b = 0, c = 0, x1 = 0, x2 = 0;
+
+    do
+    {
+        printf("Введите три числа:\n");
+        fflush(stdin);
```

Comment:
fflush is used for output streams and using it with stdin is not the best idea (because the standard doesn't specify its behavior with stdin)